tes-adapter/src/main/java/com/epam/pipeline/adaptor/TesAdapterApplication.java

```diff
@@ -0,0 +1,11 @@
+package com.epam.pipeline.adaptor;
```

Comment:
please name package `tesadapter` instead of `adaptor`

Reply:
fixed